smact/tests/test_utils.py

```
@@ -92,3 +93,73 @@ def test_convert_formula(self):
     @unittest.skipUnless(os.environ.get("MP_API_KEY"), "requires MP_API key to be set.")
     def test_download_compounds_with_mp_api(self):
         pass
+
+from smact.utils.oxidation import ICSD24OxStatesFilter
```

Ruff

Module level import not at top of file

(E402)